themes/default/css/_besocial/admin_besocial.css

```diff
@@ -103,11 +103,6 @@
 	background: #FCFCFC;
 }
 
-.theme_options .content {
-	border-top: 1px solid #CCCCCC;
```

Comment:
To be clear, this whole thing is duplicate, the same borders are defined with the same selectors up above.